packages/frontend/templates/package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "@affine/templates",
   "private": true,
-  "sideEffect": false,
-  "exports": {
-    "./*.md": "./*.md",
-    "./v1/*.json": "./v1/*.json",
-    "./preloading.json": "./preloading.json"
+  "sideEffect": true,
+  "version": "0.11.0",
+  "scripts": {
+    "gen-templates": "node ./build.mjs"
```

Comment:
can we use postinstall to gen this file and ignore it, not commit to repo.

Reply:
I tried using postinstall but it seems the file does not get generated after `yarn install`. I will take a look

Reply:
It seems yarn will only run postinstall when the dependency changed. So that's why I added the template to git.